From 915ddfc7667bc2ad50a2037205a9bebf2a47d718 Mon Sep 17 00:00:00 2001
From: SangeethaRao <sangeetha.rao@intel.com>
Date: Tue, 21 Jul 2015 15:46:32 -0500
Subject: [PATCH 41/86] dts: drivers/usb/dwc3: Added Axxia X9 xHCI DWC3 USB
 support for simulation (continue)

Signed-off-by: SangeethaRao <sangeetha.rao@intel.com>
[Original patch is from linux-yocto-4.1 axxia/base branch.]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/arm64/boot/dts/intel/axm5604-sim.dts |    4 ++++
 arch/arm64/boot/dts/intel/axm56xx.dtsi    |   12 +++++++++---
 2 files changed, 13 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/intel/axm5604-sim.dts b/arch/arm64/boot/dts/intel/axm5604-sim.dts
index f5f6063..6dbfa70 100644
--- a/arch/arm64/boot/dts/intel/axm5604-sim.dts
+++ b/arch/arm64/boot/dts/intel/axm5604-sim.dts
@@ -150,3 +150,7 @@
 &pci2 {
 	status = "okay";
 };
+
+&usb0 {
+	status = "okay";
+};
diff --git a/arch/arm64/boot/dts/intel/axm56xx.dtsi b/arch/arm64/boot/dts/intel/axm56xx.dtsi
index 376a82f..a41bf91 100644
--- a/arch/arm64/boot/dts/intel/axm56xx.dtsi
+++ b/arch/arm64/boot/dts/intel/axm56xx.dtsi
@@ -111,11 +111,17 @@
 		};
 
 		usb0: usb@9000000000 {
-			compatible = "synopsys,dwc3";
-			reg = <0x90 0x00000000 0x10000>;
-			interrupts = <GIC_SPI 90 IRQ_TYPE_LEVEL_HIGH>;
+			compatible = "intel,axxia-dwc3";
 			dma-coherent;
 			status = "disabled";
+			#address-cells = <2>;
+			#size-cells = <1>;
+			ranges;
+			dwc3 {
+				compatible = "snps,dwc3";
+				reg = <0x90 0x00000000 0x10000>;
+				interrupts = <GIC_SPI 90 IRQ_TYPE_LEVEL_HIGH>;
+			};
 		};
 
 		sata0: sata@a000000000 {
-- 
1.7.5.4

